android/src/main/java/com/evollu/react/fcm/FIRMessagingModule.java

```diff
@@ -54,12 +60,38 @@ public void requestPermissions(){
     }
 
     @ReactMethod
+    public void getInitFCMData(Promise promise) {
```

Comment:
With the current code are you able to retrieve notification data if a notification is received while app is in background ? I couldn't do that in Android that's why I resorted to this. I might be missing something though.